core/templates/pages/learner-dashboard-page/learner-groups-tab.component.html

```diff
@@ -1,86 +1,62 @@
 <div class="learner-groups-tab" [ngStyle]="!windowIsNarrow && {'margin-top': '64px'}">
   <div class="learner-of-learner-groups-section">
-    <h3 class="learner-groups-section-heading">
+    <p class="learner-groups-section-heading">
```

Comment:
Why use "p" for a heading? It seems semantically incorrect.

Reply:
Done as earlier.